.github/workflows/backport-to-release-branch.yml

```diff
@@ -0,0 +1,215 @@
+# This workflow moves marked commits from a development branch to a release branch.
+#
+# Each commit in the development branch is cherry-picked to the release branch if the commit originates from a merged
+# PR that is marked for backport. If a merge conflict occurs, the commit is committed to a new branch with merge
+# markers and then a PR is created into the target branch with those markers. The PR is labeled with
+# "type:release-merge-conflict" to indicate that it needs manual resolution.
+#
+# The PR is expected to fail compilation and status checks (of course) due to the merge conflict markers. A human
+# should then checkout the PR branch, resolve the conflicts, and push the changes back to the PR branch.
+#
+# This workflow is currently being tested in the Mu Basecore repository. Once it performs well over a reasonable
+# period of time, it will be moved to the Mu DevOps repository for general use and synced to other relevant
+# repositories using a more generalized template to fill in some of the details.
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+name: Backport Commits to Release Branch
+
+on:
+  push:
+    branches:
+      - dev/202405

```

Comment:
Needs permissions - contents:write i think

Reply:
All write operations will use the "CHERRY_PICK_TOKEN" which has write permission from the Project Mu bot account. Git operations use the token assigned in the checkout step and GitHub PR operations use the token explicitly provided in the auth header.